howtos/layers/image.md

```diff
@@ -101,11 +88,11 @@ in a zarr file:
 
 +++
 
-![image: lattice light sheet microscopy](../assets/tutorials/LLSM.gif)
+![image: lattice light sheet microscopy](../../images/LLSM.gif)
 
 +++
 
-## image pyramids
+## Image pyramids
```

Comment:
Maybe rename this section to "Multiscale image pyramids"? I think that is more self-explanatory than just image pyramids.